content/docs/operating/security.md

```diff
@@ -19,6 +19,11 @@ prometheus-team@googlegroups.com. We will fix the issue and coordinate a
 release date with you, acknowledging your effort and mentioning you by name
 if you want.
 
+Prometheus is maintained by volunteers, not by a company. Therefore, fixing on
```

Comment:
`fixing on security issues` should perhaps just be `fixing security issues`.

Reply:
fixed